### ISSUE TITLE
package ‘ggvis’ is needed and not mentioned as dependency

### ISSUE THREAD

**Issue by diegopenhanut** [friend1ws/pmsignature#10]
Hello :)

Just a tip to update the docs.
To be able to install your package I needed to install ggvis. I am using a mac (Yosemite 10.10.5) and a Centos 7.0 running R 3.2.1.
Could you add ggvis to the list of dependencies?

**Comment by friend1ws** [comment]
Hi, Thank you very much (sorry for the late response).
This is what I haven't expected, and I will modify it very soon.

**Comment by diegopenhanut** [comment]
I can update the description file and test if the package can be installed. Just let me know.

**Comment by friend1ws** [comment]
Yes, thanks. I actually modified it (at the master branch), and I'm happy if you could check it again.

This is caused by the "@import ggvis" in the visualizeSignature2.R script (where I want to test interactive plot), and auto-generated NAMESPACE.
The package ggvis is only necessary if you want tot use the function in visualizeSignature2.R.

**Comment by diegopenhanut** [comment]
I tried to install on a virtual machine (Centos7) on a clean install of R. I needed to instal dplyr to be able to install your package. Maybe because my VM was a minimum install, I needed to install curl, libcurl, libcurl-devel, libxm12, libxm12-devel to be able to Install RCurl and XLM.
